_docs-sources/pipelines/core-concepts/index.md

```diff
@@ -0,0 +1,27 @@
+# Core concepts
+
+## Overview
+
+Fundamentally, Gruntwork Pipelines is designed to roll out an _infrastructure change._ Doing that in a secure and responsible way requires understanding a few key concepts.
+
+## Concepts
+
+### Infrastructure change
+
+When you edit any infrastructure-as-code that needs to be "applied" to your cloud account (e.g. AWS or GCP), you are making an _infrastructure change_. We sometime call these an "infra-change" for short.
+
+Infra-changes can involve updates to OpenTF (Terraform) code, Packer templates, Docker files, Kubernetes Helm charts, or any other type of file that represents a desired state of your infrastructure and that needs to be somehow applied. A classic example is changing some variables on an instance of a Terraform module. By changing these variables you are essentially requesting that some API calls eventually be made to AWS to update your infrastructure in some way (in this case, by running `terraform apply` at some point).
```

Comment:
Currently we only support Terraform. None of these other tools are supported.

Reply:
This is a conceptual point, though we should clarify explicitly that only Terraform files are supported today.